Azure/sonic-buildimage
# Fix build errors in branch 201712

**- What I did**
Patch [#1755](https://github.com/Azure/sonic-buildimage/pull/1755), [#1666](https://github.com/Azure/sonic-buildimage/pull/1666) and [#1633](https://github.com/Azure/sonic-buildimage/pull/1633) from master branch to 201712 branch

**- How I did it**
Patch the files in #1755, #1666 and #1633 

**- How to verify it**
Locally patch these files and build image, verify the build process is finished successfully

**- Description for the changelog**
Fix build errors in branch 201712
